app/accept_invite/rest.py

```diff
@@ -38,6 +38,9 @@ def validate_invitation_token(invitation_type, token):
                   ['Your invitation to GOV.UK Notify has expired. '
                    'Please ask the person that invited you to send you another one']}
         raise InvalidRequest(errors, status_code=400)
+    except BadData:
+        errors = {'invitation': 'Something’s wrong with this link. Make sure you’ve copied the whole thing.'}
```

Comment:
can we achieve this by just changing `BadSignature` to `BadData` in `app/__init__.py`? (and updating the error message there.

That means we don't have multiple messages for all the different tokens - which is a list that i think looks something like this:

* '/invitation/'
* '/organisation-invitation/'
* '/new-password/'
* '/verify-email/'
* '/user-profile/email/confirm/'
* '/email-auth/'

Reply:
oh wait this is on api

Reply:
yeah I was sufficiently confused by the comment :P

Reply:
so is this needed anymore? https://github.com/alphagov/notifications-admin/blob/master/app/__init__.py#L522-L526